src/ray/gcs/accessor.h

```diff
@@ -156,7 +156,7 @@ class JobInfoAccessor {
 /// task information in the GCS.
 class TaskInfoAccessor {
  public:
-  virtual ~TaskInfoAccessor() {}
+  inline virtual ~TaskInfoAccessor() {}
```

Comment:
combining inline with virtual makes no sense (virtual functions can't be inlined)